release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2678,6 +2678,31 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-15-6"]
+=== RHSA-2024:1559 - {product-title} 4.15.6 bug fix and security update
+
+Issued: 2024-04-02
+
+{product-title} release 4.15.6, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:1559[RHSA-2024:1559] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1563[RHSA-2024:1563] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.6 --pullspecs
+----
+
+[id="ocp-4-15-6-known-issues"]
+==== Known issue
+
+* There is a known issue in this release which causes the `oc-mirror` binary to fail on Red Hat Enterprise Linux (RHEL) 8 systems. We recommend using the Red Hat OpenShift Container Platform 4.15.5 `oc-mirror` binary, or extracting `oc-mirror.rhel8` for this specific case. (link:https://issues.redhat.com/browse/OCPBUGS-31609[*OCPBUGS-31609*])
```

Comment:
> We recommend using ...

**issue:** https://redhat-documentation.github.io/supplementary-style-guide/#recommend 
Instead of "Red Hat recommends", direct users to take the recommended action. 

Maybe change this to:
```
Use the Red Hat OpenShift Container Platform 4.15.5 `oc-mirror` binary 
or extract `oc-mirror.rhel8` for this case.
```